src/test/java/org/folio/service/users/UsersLookUpServiceTest.java

```diff
@@ -145,24 +143,6 @@ public void shouldReturn404WhenUserNotFound(TestContext context) {
     });
   }
 
-  @Test
-  public void shouldReturn500WhenMissingOkapiURLHeader(TestContext context) {
```

Comment:
The test was removed because the check for existed okapiUrl header performed in OkapiParams. So there is no chance that okapiParams comes to the service without that header.